<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ShopRedesign04" name="com.tcs.BsiShopRedesign.sprint4_5.RegistrationsWithDiffProfile" tests="5" failures="1" timestamp="18 Sep 2018 16:33:31 GMT" time="57.892" errors="0">
  <testcase name="verifyLoginForRegisteredUser" time="13.413" classname="com.tcs.BsiShopRedesign.sprint4_5.RegistrationsWithDiffProfile"/>
  <testcase name="loginThroughAlreadyRegistered" time="6.723" classname="com.tcs.BsiShopRedesign.sprint4_5.RegistrationsWithDiffProfile">
    <failure type="java.lang.AssertionError" message="Verify Login Error Message for Registered User was unsuccessful">
      <![CDATA[java.lang.AssertionError: Verify Login Error Message for Registered User was unsuccessful
	at org.testng.Assert.fail(Assert.java:94)
	at com.tcs.BsiShopRedesign.utilities.CommonHelper.reportFailure(CommonHelper.java:66)
	at com.tcs.BsiShopRedesign.pages.HomePage.loginThroughAlreadyRegistered(HomePage.java:655)
	at com.tcs.BsiShopRedesign.sprint4_5.RegistrationsWithDiffProfile.loginThroughAlreadyRegistered(RegistrationsWithDiffProfile.java:143)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- loginThroughAlreadyRegistered -->
  <testcase name="verifyTooltip" time="16.633" classname="com.tcs.BsiShopRedesign.sprint4_5.RegistrationsWithDiffProfile"/>
  <testcase name="verifyContactInformation" time="7.245" classname="com.tcs.BsiShopRedesign.sprint4_5.RegistrationsWithDiffProfile"/>
  <testcase name="createRegUserUKProfile" time="13.878" classname="com.tcs.BsiShopRedesign.sprint4_5.RegistrationsWithDiffProfile"/>
</testsuite> <!-- com.tcs.BsiShopRedesign.sprint4_5.RegistrationsWithDiffProfile -->
